modules/actions/app.js

```diff
@@ -185,3 +186,14 @@ export function clearSearchState() {
 export function clearCurrentApp() {
 	return createAction(AppConstants.APP.CLEAR_CURRENT_APP);
 }
+
+export function updateAppPaymentMethod(stripeToken, product) {
```

Comment:
I believe this is going to be an update method for clusters & arc clusters, so you can rename this to to `updateClustersPaymentMethod`

Reply:
no, it takes second param product enum: `[ARC, CLUSTER, APP]`

Reply:
okay 👍